front/src/views/movies/movies.view.js

```diff
@@ -0,0 +1,21 @@
+export default {
+  name: 'ff-movies',
+  filters: {
+    modifyActorsList (titles) {
+      return titles.map(element => ' ' + element.actorName).join();
+    }
+  },
+  methods: {
+    getImgSrc (imgPath) {
+      return this.$store.state.baseUrl + '/' + imgPath;
```

Comment:
hm. might is it useful filter. i am not sure
also, you can use this syntax `${this.$store.state.baseUrl}/${imgPath}`